flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/AdaptedRestartPipelinedRegionStrategyNG.java

```diff
@@ -148,19 +145,10 @@ private RestartCallback createResetAndRescheduleTasksCallback(final long globalM
 			} catch (GlobalModVersionMismatch e) {
 				throw new IllegalStateException(
 					"Bug: ExecutionGraph was concurrently modified outside of main thread", e);
-			} catch (Exception e) {
-				throw new CompletionException(e);
-			}
-		};
-	}
-
-	private BiFunction<Object, Throwable, Object> failGlobalOnError() {
```

Comment:
This method is still needed for failure handling of the first part of failover process.